lib/service/service.go

```diff
@@ -42,6 +42,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/cloudflare/cfssl/log"
```

Comment:
Is this the log class that you want to use?

Reply:
Thanks for pointing out, i wanted to use teleport log only, will correct it